kore/test/Test/Kore/Strategies/AllPath/AllPath.hs

```diff
@@ -427,7 +432,7 @@ newtype AllPathIdentity a = AllPathIdentity { unAllPathIdentity :: Identity a }
 
 instance MonadLog AllPathIdentity where
```

Comment:
This instance should no longer be necessary.

Reply:
```
transitionRule
        :: (MonadCatch m, MonadSimplify m)
        => Prim goal
        -> ProofState goal goal
        -> Strategy.TransitionT (Rule goal) m (ProofState goal goal)
```
is used [here](https://github.com/kframework/kore/blob/0ca0bbf5e5a041dc61da252e1eb97c6ae1299523/kore/test/Test/Kore/Strategies/AllPath/AllPath.hs#L423), and `MonadSimplify` requires `MonadLog`. Does it mean it's still necessary?

Reply:
Yes, it is.